src/components/JoinCommunity/index.jsx

```diff
@@ -26,6 +26,7 @@ export default function Community() {
                     {/* <LinkCard link="#" title="Slack" light="img/community_links/slack.svg" dark="img/community_links/slack.svg" /> */}
                     <LinkCard link="https://github.com/uptane" title="GitHub" light="img/community_links/github_light.svg" dark="img/community_links/github_dark.svg" />
                     <LinkCard link="https://discord.gg/uWyT6gDCqx" title="Discord" light="img/community_links/discord.svg" dark="img/community_links/discord.svg" />
+                    <LinkCard link="https://www.linkedin.com/company/uptane/" title="Linkedin" light="img/community_links/Linkedin_logo.svg" dark="img/community_links/Linkedin_logo.svg" />
```

Comment:
lower case the first letter of `Linkedin_logo.svg` it should be `linkedin_logo.svg` same as stored in `img/community_links/Linkedin_logo.svg` folder as if we ignore this it will drop an error during the build process, as it is case sensitive